### PR TITLE
Update info.xml and add screenshot

### PR DESCRIPTION
**Pull request by dlandon**
Update info.xml and add screenshot and prepare app for marketplace.

### CODE REVIEW

**Comment by dlandon** [comment]
Sorry for all the gyrations.  I'm not familiar with collaborative github projects.  I think I finally have it worked out.

File: appinfo/info.xml [hunk 1]
```diff
@@ -6,9 +6,12 @@
 	<namespace>Tasks</namespace>
 	<category>productivity</category>
 	<licence>AGPL</licence>
-	<author>Raimund Schlüßler</author>
-	<description>Task App for Calender TODOs</description>
-	<ocsid>164356</ocsid>
+	<author>Dan Landon</author>
+	<summary>ownCloud Tasks App for Calender</summary>
+	<description>Once enabled, a new Tasks menu will appear in your ownCloud apps menu. From there you can add and delete tasks, edit their title, description, start and due dates, reminder times, mark them as important, and add comments on them. Tasks can be shared between users. Tasks can be synchronized using CalDav (each task list is linked to an ownCloud calendar, to sync it to your local client - Thunderbird, Evolution, KDE Kontact, iCal... - just add the calendar as a remote calendar in you client). You can download your tasks as ICS files using the download button for each calendar.</description>
+	<screenshot>https://github.com/owncloud/tasks/raw/master/screenshots/screenshot.png</screenshot>
```
Comment (DeepDiver1975) [inline appinfo/info.xml]: link is wrong - gives 404 - might be different after merge .....

Comment (DeepDiver1975) [reply appinfo/info.xml]: you could also use this https://cloud.githubusercontent.com/assets/2496460/9783697/ab337634-57a6-11e5-9633-b8f950b1e28e.png

Comment (dlandon) [reply appinfo/info.xml]: The link will work after merge.

I will update the screenshot.

**Comment by dlandon** [comment]
Is there something else I need to do to get this pull request merged?

**Comment by dlandon** [comment]
Can someone please review this pull request so I can get the app posted on the market?

File: appinfo/info.xml [hunk 2]
```diff
@@ -6,9 +6,12 @@
 	<namespace>Tasks</namespace>
 	<category>productivity</category>
 	<licence>AGPL</licence>
-	<author>Raimund Schlüßler</author>
-	<description>Task App for Calender TODOs</description>
-	<ocsid>164356</ocsid>
+	<author>Dan Landon</author>
```
Comment (DeepDiver1975) [inline appinfo/info.xml]: you should really keep the original author - feel free to add yourself. But removing Raimund is not fair

Comment (dlandon) [reply appinfo/info.xml]: Ok.

**Pull request merged**